packages/lib/utils/chunkGenerator.ts

```diff
@@ -0,0 +1,49 @@
+import { read, open, stat } from 'fs';
+import { promisify } from 'util';
+
+const readPromisify = promisify(read);
+const statPromisify = promisify(stat);
+const openPromisify = promisify(open);
```

Comment:
Please remove and check `fs/promise` - all those are already available as async functions. It was 4 or 5 years ago that we needed to promisify functions but today that's not needed